config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
```

Comment:
- [ ] Great job adding the routes but I think as mentioned by the previous reviewer when I open your app I see just a blank page but I should see the login page instead of a blank page so kindly fix your routes 💯🔥✨🚀